### DISCUSSION TITLE
Windows系统下如何将本机存储的根目录设置为此电脑？

### DISCUSSION THREAD

**Discussion by VaporTang** [AlistGo/alist#5194] [Q&A]
请问我应该如何设置本地根目录以实现alist挂载"此电脑"的效果，也就是说，整个电脑的所有分区（C D E等）都是作为同一个存储的，而不是设置多个"本机存储"

**Comment by xhofe** [comment]
暂不支持

**Reply by VaporTang** [reply]
有空可以考虑支持下吗, 这样的跨分区移动文件会方便些
感谢作者
